libs/shared/data-access-strapi/src/lib/page.service.ts

```diff
@@ -88,6 +92,11 @@ export class BaPageService<T = any> {
       .get<T>(requestPath, { responseType: 'json' })
       .pipe(tap((data) => this._cache.set(id, data)));
   }
+
+  private _fetchRoutes(): Observable<T> {
+    const requestPath = `data/routes.txt`;
+    return this._http.get<T>(requestPath, { responseType: 'json' });
+  }
```

Comment:
Why do we need this information in there?

Reply:
Left over from something I tried. Getting the routes for the top nav but found a better solution I want to implement soon